test/rubocop/git/commit_file_test.rb

```diff
@@ -0,0 +1,44 @@
+require_relative '../../test_helper'
+require 'rubocop/git/commit_file'
+
+describe RuboCop::Git::CommitFile do
+  describe 'ruby files check' do
```

Comment:
I'd keep all this in a single test. I like the one "logical" assertion approach. It also avoids the double describes.